lib/Model/Listing/Dao/QueryBuilderHelperTrait.php

```diff
@@ -23,13 +23,26 @@
 trait QueryBuilderHelperTrait
 {
     /**
-     * @var callable|null
+     * @var callable[]
      */
-    protected $onCreateQueryBuilderCallback;
+    protected array $queryBuilderProcessors = [];
```

Comment:
oh wait, this might be a bc-break, the trait is not internal

Reply:
emh technically yes, but on paper, it doesn't have this doc block
https://github.com/pimcore/pimcore/blob/df14e025d3db615fc75b190e3c9f3cc1d68d7304/lib/Controller/Traits/ElementEditLockHelperTrait.php#L20-L22
so i am not 100% sure if it's safe to change, even though the risk look extremely minimal and rare

Reply:
let's see if @fashxp can have a quick look and confirm the changes are safe